src/coreclr/gc/gc.h

```diff
@@ -374,7 +374,7 @@ inline bool IsServerHeap()
 HRESULT initialize_log_file();
 void flush_gc_log (bool);
 void GCLog (const char *fmt, ... );
-#define dprintf(l,x) {if ((l == 1) || (l == GTC_LOG)) {GCLog x;}}
+#define dprintf(l,x) {if ((l == 6666)) {GCLog x;}}
```

Comment:
Let's revert all non-Reliability framework related changes to the PR. I believe they are impertinent to the changes you are trying to make to the RF.